docs/scos/user/technology-partners/202108.0/payment-partners/bs-payone/legacy-demoshop-integration/payone-facade.md

```diff
@@ -14,42 +14,45 @@ redirect_from:
+| `isPaymentUnderpaid` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Underpaid" status for a certain order. |
+| `isPaymentRefund` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Refund" status for a certain order. |
+| `isPaymentAppointed` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Appointed" status for a certain order. |
+| `isPaymentOther` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the status different from "Paid", "Underpaid" or "Appointed" for a certain order. |
+| `isPaymentCapture` | Order Id, Order Item Id | bool | Checks if there is an unprocessed transaction status update from PayOne with the "Capture" status for a certain order. |
+| `postSaveHook` (deprecated) | `QuoteTransfer`, `CheckoutResponseTransfer` | `CheckoutResponseTransfer` | Handles redirects and errors after an order is placed. |
+| `orderPostSave` | `QuoteTransfer`, `CheckoutResponseTransfer` | `CheckoutResponseTransfer` | Handles redirects and errors after an order is placed. Depending on the payment method, it executes the authorization or pre-authorization API call. Updates `CheckoutResponseTransfer` with errors or/and redirect URL accordingly to the API response. |
+| `getPaymentLogs` | `ObjectCollection` | array of `PayOnePaymentLogTransfer` | Gets payment logs (both api and transaction status) for specific orders in chronological order. |
 | `getPaymentDetail` | Order Id | `PaymentDetailTransfer` | Gets payment details for a specific order. |
 | `updatePaymentDetail` | `PaymentDetailTransfer`, Order Id | void | Updates payment details for a specific order. |
 | install | `MessengerInterface` | void | Installs module translations to project glossary. |
-| `startPaypalExpressCheckout` | `PayoneStartPaypalExpressCheckoutRequestTransfer` | `PayonePaypalExpressCheckoutGenericPaymentResponseTransfer` | Sends request to payone in order to start express checkout and get the workorderid which is used for further operations. |
-| `getPaypalExpressCheckoutDetails` | `QuoteTransfer` | `PayonePaypalExpressCheckoutGenericPaymentResponseTransfer` | Returns customer specific information retrieved from paypal through payone (e.g. email and shipping info). |
+| `startPayPalExpressCheckout` | `PayOneStartPayPalExpressCheckoutRequestTransfer` | `PayOnePayPalExpressCheckoutGenericPaymentResponseTransfer` | Sends request to PayOne in order to start express checkout and get the workorderid which is used for further operations. |
+| `getPayPalExpressCheckoutDetails` | `QuoteTransfer` | `PayOnePayPalExpressCheckoutGenericPaymentResponseTransfer` | Returns customer specific information retrieved from PayPal through PayOne (e.g., email and shipping info). |
```

Comment:
e.g.-> for example (https://developers.google.com/style/abbreviations?hl=en#dont-use)